config/ecs/otel-agent-ecs-default-config.yaml

```diff
@@ -0,0 +1,39 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:55680
+      http:
+        endpoint: localhost:55681
+  awsxray:
+    endpoint: 0.0.0.0:2000
+    transport: udp
+
+processors:
+  batch/traces:
+    timeout: 1s
+    send_batch_size: 50
+  batch/metrics:
+    timeout: 60s
+
+exporters:
+  awsxray:
+  awsemf:
+    log_group_name: aws/ecs/containerinsights/{aws.ecs.cluster.name}/application/metrics
+    log_stream_name: { aws.ecs.task.id }
```

Comment:
this is what I was expecting, I am guessing the other file with the app caps ECS in the file name must have been committed on accident...

Reply:
Yes, sorry.